docs/source/howto/installation.rst

```diff
@@ -18,7 +18,149 @@ Managing profiles
+AiiDA will go through each of the paths and check whether they contain a configuration directory, i.e., a folder with the name ``.aiida``.
+The first configuration directory that is encountered will be used as the configuration directory.
+If no configuration directory is found, one will be created in the last path that was considered.
+For example, the directory structure in your home folder ``~/`` might look like this::
+
+    .
+    ├── .aiida
+    └── project_a
+        ├── .aiida
+        └── subfolder
+
+If you leave the ``AIIDA_PATH`` variable unset, the default location ``~/.aiida`` will be used.
+However, if you set:
+
+.. code:: bash
+
+    export AIIDA_PATH='~/project_a:'
+
+the configuration directory ``~/project_a/.aiida`` will be used.
+
+.. warning::
+    If there was no ``.aiida`` directory in ``~/project_a``, AiiDA would have created it for you, so make sure to set the ``AIIDA_PATH`` correctly.
+
+
+.. _how-to:installation:configure:tab-completion:
+
+Activating tab-completion
+-------------------------
+The ``verdi`` command line interface has many commands and parameters, which can be tab-completed to simplify its use.
+To enable tab-completion, the following shell command should be executed:
+
+.. code:: bash
+
+    eval "$(_VERDI_COMPLETE=source verdi)"
+
+Note that this *only* enables tab-completion in the *current shell*.
+In order to make this change permanent and have it automatically applied to any shell instance, you can add the command to the start up script.
+This file is shell specific, but likely one of the following:
+
+    * the startup file of your shell (``.bashrc``, ``.zsh``, ...), if aiida is installed system-wide
+    * the `activation script <https://virtualenv.pypa.io/en/latest/userguide/#activate-script>`_ of your virtual environment
+    * a `startup file <https://conda.io/docs/user-guide/tasks/manage-environments.html#saving-environment-variables>`_ for your conda environment
+
+
+.. important::
+
+    After you have added the line to the start up script, make sure to restart the terminal or source the script for the changes to take effect
+
+
+.. _how-to:installation:configure:options:
+
+Configuring profile options
+---------------------------
+In addition to the configuration of AiiDA instances described in the previous sections, AiiDA also provides various configurational options for profiles.
+To control the configuration options of profiles use the :ref:`verdi config<reference:command-line:verdi-config>` command.
+To set a configurational option, simply pass the name of the option and the value to set ``verdi config OPTION_NAME OPTION_VALUE``.
+The available options are tab-completed, so simply type ``verdi config`` and thit <TAB> twice to list them.
+
+For example, if you want to change the default number of workers that are created when you start the daemon, you can run::
+
+.. code:: bash
+
+    (aiida): verdi config daemon.default_workers 5
```

Comment:
Do we already have a convention as to how to present a shell prompt? I believe that for the tutorial we have decided to use `$`.

Reply:
Good point. Using the `$` is fine. In some places prepending a virtual environment is useful to remind people they should load it first. Then again, it may look differently for different virtual environment types, people might intentionally decide to install system wide. On top of that, we would have to use this everywhere in the documentation, even in places where this might be considered self-evident. I guess I am just making the point for myself that I will switch to just the `$` as prompt ^^